### ISSUE TITLE
Parent theme support

### ISSUE THREAD

**Issue by ghost** [Brugman/acf-agency-workflow#3]
Firstly great plugin, thank you.

I have a scenario where fields are saved in parent theme, and the child theme just uses these fields. When changes are made, they are saved in parent theme.

Your plugin is exactly what i was looking for as i used custom fields in a custom plugin also. However whilst i can use Plugin and Child theme together, i cant get Plugin and Parent theme to work. Creating an acf-json folder in parent theme doesn't do anything.

I can use the filters mentioned here

https://support.advancedcustomfields.com/forums/topic/acf-json-fields-not-loading-from-parent-theme/

but that forces saves inside parent theme and my plugin fields are then ignored (which defeats the purpose. I found your plugin to solve this very issue)

TLDR, can you detect acf-json from parent theme also?

**Comment by Brugman** [comment]
Sounds good. I'll give it a shot.

**Comment by Brugman** [comment]
Can you give 1896bef2f348bf37e3780bf0bb35030ab47aba09 a test?
There's a new [parent theme snippet](https://github.com/Brugman/acf-agency-workflow#set-an-acf-local-json-location-in-a-parent-theme) in the readme.

**Comment by ghost** [comment]
fantastic. Works like a charm.

This plugin is amazing. can i suggest pushing to WP repo? This plugin is the biggest enhancement to ACF that i have implemented into my workflow since ACF itself. opens up a lot of avenues when it comes to project development.

Personally i think this should be part of ACF itself, or consider working with ACF Extended to implement there?

Anyhow, thank you very much

**Comment by Brugman** [comment]
Cool! I do want to get my first plugin on the repo in 2020, but this one is my most complicated one, and there's parts of the code that I don't even fully understand, so this one's going to live on github for a while. I do want to look into an updating system, so people know when there's an update.

If Elliot wants to add this functionality to ACF core I'm sure he can. And I would be fine with that. There's been threads about syncing going back a few years, so I think he knows there's demand, but probably not enough.

ACF Extended I hadn't heard of before. Looks like a huge bundle, I'm getting a Jetpack-like itch. I'd have to try it out.
